### PR TITLE
Fix script CBSearch

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17654

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] CBFindIP
- [ ] Dependency 2
- [ ] Dependency 3

### CODE REVIEW

File: Scripts/script-CBSearch.yml [hunk 1]
```diff
@@ -49,7 +49,10 @@ script: |-
       };
       if(args.query)
           myArgs.query = args.query;
-      res = executeCommand('cb-' + searchType, myArgs);
+          if(searchType == 'binary')
```
Comment (Itay4) [inline Scripts/script-CBSearch.yml]:
```suggestion
          if (searchType == 'binary')
```

File: Scripts/script-CBSearch.yml [hunk 2]
```diff
@@ -49,7 +49,10 @@ script: |-
...
+              res = executeCommand('cb-' + searchType, myArgs);
```
Comment (Itay4) [inline Scripts/script-CBSearch.yml]: if we already know the the command name, no need for the string concat here, just execute the command name

**Comment by Itay4** [comment]
@kirbles19 please review yml and rn files

**Comment by kirbles19** [comment]
done.

**Pull request closed without merging**